gcp-alarm-channel/main.tf

```
@@ -0,0 +1,27 @@
+variable "md_metadata" {
+  description = "Massdriver package metadata object."
+  type        = any
+}
+
+module "apis" {
+  source   = "github.com/massdriver-cloud/terraform-modules//google-enable-apis?ref=9201b9f"
+  services = ["monitoring.googleapis.com"]
+}
+
+resource "google_monitoring_notification_channel" "main" {
+  depends_on   = [module.apis]
+  display_name = "Massdriver Alarms"
```

```suggestion
  display_name = "${var.md_metadata.name_prefix}-alarms"
```

That would follow the pattern of the other two

---

Is there a limit on `display_name` in GCP? In the other clouds its limited as its used in SMS Subjects (who does that anymore) ... the `name` field in Azure & AWS are identifiers.

---

>It is recommended that you specify a non-empty and unique name in order to make it easier to identify the channels in your project, though this is not enforced. The display name is limited to 512 Unicode characters.

So this is just the UI ... its not an actual constraint. Would you prefer the md name prefix here? Seems like we have the room and GCP doesnt have a `name` field like the other clouds.